dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -714,6 +718,21 @@ private Response multipartPUTandPOST(HttpServletRequest request, HttpServletResp
                     .equals("json")) {
                 try {
                     processJSON(contentlet, part.getEntityAs(InputStream.class));
+                    try {
+                        binaryFieldsInput =
+                            webResource.processJSON(part.getEntityAs(InputStream.class)).get("binary_fields")
+                                .toString();
+                    } catch (NullPointerException npe) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Avoid catching NullPointerException. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1696)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)